packages/server/graphql/public/mutations/updateAutoJoin.ts

```diff
@@ -0,0 +1,26 @@
+import updateTeamByTeamId from '../../../postgres/queries/updateTeamByTeamId'
+import {getUserId, isSuperUser, isTeamMember} from '../../../utils/authorization'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const updateAutoJoin: MutationResolvers['updateAutoJoin'] = async (
+  _source,
+  {teamIds, autoJoin},
+  {authToken}
+) => {
+  const viewerId = getUserId(authToken)
+  if (!isSuperUser(authToken)) {
+    const isNotMember = teamIds.some((teamId) => !isTeamMember(authToken, teamId))
+    if (isNotMember) {
+      return standardError(new Error('Not on team(s)'), {userId: viewerId})
+    }
```

Comment:
-1 Either we say we make this a manual step and skip verification for now (then it should be a private mutation) or we should ensure they're billing leader in absence of another admin like role.